r/R/csv.R

```diff
@@ -289,20 +289,20 @@ csv_convert_options <- function(check_utf8 = TRUE) {
   ))
 }
 
-#' Arrow CSV table reader
+#' Arrow CSV and JSON table readers
 #'
-#' These methods wrap the Arrow C++ CSV table reader.
+#' These methods wrap the Arrow C++ CSV and JSON table readers.
 #' For an interface to the CSV reader that's more familiar for R users, see
 #' [read_csv_arrow()]
 #'
 #' @param file A character path to a local file, or an Arrow input stream
-#' @param read_options, see [csv_read_options()]
-#' @param parse_options, see [csv_parse_options()]
-#' @param convert_options, see [csv_convert_options()]
+#' @param read_options see [csv_read_options()]
+#' @param parse_options see [csv_parse_options()]
+#' @param convert_options see [csv_convert_options()]
 #' @param ... additional parameters.
 #'
-#' @return An `arrow::csv::TableReader` R6 object. Call `$Read()` on it to get
-#' an Arrow Table.
+#' @return An `arrow::csv::TableReader` or `arrow::json::TableReader` R6
```

Comment:
I only see what look like csv methods below?

Reply:
I agree this doesn't seem right

Reply:
`json_table_reader`'s docs are merged with `csv_table_reader` here: https://github.com/apache/arrow/pull/4736/files#diff-ae58975e47ac9d8718c62ee3c1807116R64

This is what the end result looks like: https://enpiar.com/arrow-r-site/reference/csv_table_reader.html